api/src/lib/sqs.ts

```diff
@@ -0,0 +1,46 @@
+import AWS from 'aws-sdk';
+
+const queueUrl = process.env.SQS_ENDPOINT!;
```

Comment:
ESlint shows a warning on this line. I guess removing that non-null assertion will produce other TS issues. A workaround for this would be to check if that 'queueUrl' is undefined and throw an error if so (since you want to make sure that SQS_ENDPOINT env var is available), like we do on ui and e2e for env vars.